system/jlib/jptree.cpp

```diff
@@ -4331,6 +4331,20 @@ void mergePTree(IPropertyTree *target, IPropertyTree *toMerge)
     }
 }
 
+//similar to mergePTree, but replaces target attributes that also exist in toMerge
+void mergeOverwritePTree(IPropertyTree *target, IPropertyTree *toMerge)
```

Comment:
Rather than adding this I think you can just use 
`jlib_decl void mergeConfiguration(IPropertyTree & target, const IPropertyTree & source, const char *altNameAttribute=nullptr, bool overwriteAttr=true);`
It will merge child elements.

We are good as is, but we should note never to add a child to "ldap:" just called "name", that will confuse mergeConfig.  The same is true other places as well.

Reply:
Right, agreed. I missed that mergeConfiguration was already there and provides the functionality wanted.
Not worth adding mergeOverwrite at this juncture in light of that - although mergePTree is confusing and it would be good to have these function names and comments clarify their functionality.